.log.io/harvester.conf

```diff
@@ -1,7 +1,5 @@
 var config = {};
 config = Object.assign(config, require('../config.json'));
-try {
-  config = Object.assign(config, require('../config.local.json'));
-} catch (err) {}
+config = Object.assign(config, require('../config.local.json'));
```

Comment:
Removing the try/catch makes sense if the file exists, because bibliomap should not start if the local config file is invalid. However we should check for its existence, either by reading it with `fs` or by checking the code of the error catched by the try/catch (MODULE_NOT_FOUND)